packages/x-components/src/x-modules/scroll/components/__tests__/window-scroll.spec.ts

```diff
@@ -1,22 +1,37 @@
 import { mount, Wrapper } from '@vue/test-utils';
-import { getDataTestSelector, installNewXPlugin } from '../../../__tests__/utils';
-import BaseMainScroll from '../base-main-scroll.vue';
+import { getDataTestSelector, installNewXPlugin } from '../../../../__tests__/utils';
+import { XPlugin } from '../../../../plugins/x-plugin';
+import { scrollXModule } from '../../x-module';
+import WindowScroll from '../window-scroll.vue';
 
-function renderBaseMainScroll({
-  template = `<BaseMainScroll :throttleMs="throttleMs" :tag="tag"/>`,
+async function renderWindowScroll({
+  template = `<WindowScroll :throttleMs="throttleMs" :tag="tag"/>`,
   throttleMs = 200,
   scrollHeight = 800,
   clientHeight = 200,
   distanceToBottom = 100,
   tag,
```

Comment:
You can use here a default `body` or `html` and remove the half of `tag` in the calls to the `renderBaseMainScroll` function.